frontend/src/App.scss

```diff
@@ -0,0 +1,15 @@
+@import "Variables";
+
+.App {
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  min-width: 100vw;
+  background-color: $background-color2;
```

Comment:
Using variables is very nice, but be sure to give them descriptive names. Reading only this without checking the variables file you will have absolutely no idea what color this is.